src/screens/GroupSetting/hooks/useGroupSetting.js

```diff
@@ -141,7 +174,7 @@ const useGroupSetting = ({navigation, route}) => {
     isLoading,
     setIsLoading,
     handleResLaunchGallery,
-    updateDataEdit
+    loadingUpdate
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `loadingUpdate` state variable is now being returned from the hook. This is a reasonable change as long as this variable is being used in the component that uses this hook.

```diff
- updateDataEdit
+ loadingUpdate
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->